fpa/noopauditer/noopauditer.go

```diff
@@ -0,0 +1,35 @@
+package noopauditer
```

Comment:
`noopauditer` is an `Auditer` that doesn't save any of the events that are reported for a CID. Mostly for tests at the moment.